lib/gui/modules/image-writer.js

```diff
@@ -102,12 +113,32 @@ imageWriter.service('ImageWriterService', function($q, $rootScope) {
         flashState.setProgressState(state);
       });
 
-    }).then(flashState.unsetFlashingFlag).catch((error) => {
+    }).then(flashState.unsetFlashingFlag).then(() => {
+      if (flashState.wasLastFlashCancelled()) {
+        AnalyticsService.logEvent('Elevation cancelled', analyticsData);
```

Comment:
I guess theoretically, all these logEvents only need to report the uuid, since we can get all the other details from the 'Flash' event with the matching uuid?